site/src/pages/tutorials/thrift/blog/run-service.mdx

```diff
@@ -0,0 +1,144 @@
+
+```java filename=BlogServiceImpl.java
+package example.armeria.server.blog.thrift;
+
+import example.armeria.blog.thrift.BlogService;
+
+public class BlogServiceImpl implements BlogService.AsyncIface {}
+```
+
+## 2. Override service methods
+
+Add empty service methods as follows to override the service methods.
+We'll implement the service methods one by one in this tutorial.
+Let's leave them empty for now.
+
+```java filename=BlogServiceImpl.java
+import example.armeria.blog.thrift.BlogPost;
+import example.armeria.blog.thrift.CreateBlogPostRequest;
+import example.armeria.blog.thrift.GetBlogPostRequest;
+import example.armeria.blog.thrift.ListBlogPostsRequest;
+import example.armeria.blog.thrift.ListBlogPostsResponse;
+import example.armeria.blog.thrift.UpdateBlogPostRequest;
+import example.armeria.blog.thrift.DeleteBlogPostRequest;
+import org.apache.thrift.TException;
+import org.apache.thrift.async.AsyncMethodCallback;
+
+public class BlogServiceImpl implements BlogService.AsyncIface {
+
+  @Override
+  public void createBlogPost(CreateBlogPostRequest request, AsyncMethodCallback<BlogPost> resultHandler)
+          throws TException {}
+
+  @Override
+  public void getBlogPost(GetBlogPostRequest request, AsyncMethodCallback<BlogPost> resultHandler)
+          throws TException {}
+
+  @Override
+  public void listBlogPosts(ListBlogPostsRequest request, AsyncMethodCallback<ListBlogPostsResponse> resultHandler)
+          throws TException {}
+
+  @Override
+  public void updateBlogPost(UpdateBlogPostRequest request, AsyncMethodCallback<BlogPost> resultHandler)
+          throws TException {}
+
+  @Override
+  public void deleteBlogPost(DeleteBlogPostRequest request, AsyncMethodCallback<Void> resultHandler)
+          throws TException {}
+}
+```
+
+## 3. Create a server
+
+Let's create a server to serve our service.
+
+1. Create the `Main` class for the server.
+You can see the full version of the file [here](https://github.com/line/armeria-examples/tree/main/tutorials/thrift/src/main/java/example/armeria/server/blog/thrift/Main.java).
+  ```java filename=Main.java
+  package example.armeria.server.blog.thrift;
+
+  import org.slf4j.Logger;
+  import org.slf4j.LoggerFactory;
+
+  public final class Main {
+    private static final Logger logger = LoggerFactory.getLogger(Main.class);
+  }
+  ```
+2. Add the `newServer()` method in the `Main` class as follows.
+We are using Armeria's <type://THttpService> to handle Thrift calls.
+  ```java filename=Main.java
+  import com.linecorp.armeria.server.Server;
+  import com.linecorp.armeria.server.thrift.THttpService;
+  ...
+  private static Server newServer(int port) throws Exception {
+    final THttpService tHttpService =
+              THttpService.builder()
+                          .addService(new BlogServiceImpl())
```

Comment:
To other maintainers, I'd rather the code in the tutorial be maintainable rather than all perfectly indented.
If needed, I think we can apply a certain code formatting rule across the entire site instead of handling each case manually. e.g. https://github.com/line/armeria/pull/5486
Let me know what you think 🙇